app/src/main/java/com/github/tatsuyafujisaki/githubactionssample/MainActivity.kt

```diff
@@ -9,6 +9,8 @@ import kotlinx.android.synthetic.main.activity_main.*
 
 class MainActivity : AppCompatActivity() {
 
+    val aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa = "test"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tatsuyafujisaki/android-github-actions-firebase-danger-sample/blob/974140241f9ef867a7f133755ca32f49c9f42060/app/src/main/java/com/github/tatsuyafujisaki/githubactionssample/MainActivity.kt#L12"></span>Exceeded max line length (100) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>